app/src/components/sections/SingleTeamView/MemberSection.vue

```diff
@@ -121,4 +127,120 @@ const searchUsers = async (input: { name: string; address: string }) => {
     addErrorToast(error.message)
   }
 }
+
+const {
+  execute: executeFetchRoleCategories,
+  data: _roleCategories
+} = useCustomFetch('role-category', {
+  immediate: false
+})
+  .get()
+  .json()
+
+const createContract = async (member: Partial<MemberInput>) => {
+  let contract
+  if (member.roles)
```

Comment:
i think this check should be like this right?
```suggestion
  if (member.roles.length > 0)
```

because empty array will return `true`